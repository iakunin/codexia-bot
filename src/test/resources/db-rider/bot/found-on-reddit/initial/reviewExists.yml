github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

codexia_project:
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 12
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

github_repo_source:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "12"
    -
        id: 2
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "REDDIT"
        external_id: "9xsr9i"

codexia_review:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        codexia_project_id: 5
        author: "FOUND_ON_REDDIT"
        reason: "9xsr9i"
        text: >-
            This project is found on Reddit:
            [web link](https://www.reddit.com/comments/9xsr9i),
            [api link](https://www.reddit.com/api/info.json?id=t3_9xsr9i).
