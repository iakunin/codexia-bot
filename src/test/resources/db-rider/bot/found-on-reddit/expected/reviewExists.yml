codexia_review:
    -
        id: 1
        codexia_project_id: 5
        author: "FOUND_ON_REDDIT"
        reason: "9xsr9i"
        text: >-
            This project is found on Reddit:
            [web link](https://www.reddit.com/comments/9xsr9i),
            [api link](https://www.reddit.com/api/info.json?id=t3_9xsr9i).
